wiki/Tournaments/TEMPLATE.md

```diff
@@ -0,0 +1,148 @@
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") |  |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") |  |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") |  |
+| *4th place* |  |
+
+![](img/badge.jpg "{SHORT_TITLE} winner badge") <!-- remove if not available -->
+
+## Organisation
+
+The {TITLE} is run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager |  |
+| Mappool selector |  |
+| Streamer |
+| Commentator |  |
+| Designer |
+| Referee |
+| Statistician |  |
+<!-- | Example | ![][flag_AU] [peppy](https://osu.ppy.sh/users/2), ![][flag_AU] [BanchoBot](https://osu.ppy.sh/users/3) | -->
+
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/{TOPIC_NUM})
+- [Livestream](https://www.twitch.tv/AccountName "Twitch")
+- **[Statistics sheet](StatisticsLink "App Name")**
+
+## Participants
+
+<!-- for team/country-based tournaments -->
+
+|  | Country | Members |
+| :-: | :-: | :-- |
+| ![][flag_AU] | **Australia** | [peppy](https://osu.ppy.sh/users/2), [BanchoBot](https://osu.ppy.sh/users/3) |
+|  |  |  |
+
+<!-- for 1v1 tournaments -->
+
+| Seed | Members |
+| :-- | :-- |
+| Top |  |
+| Middle |  |
+
+<!-- remove the following section if seeding was displayed above -->
+
+## Groups
+
+| Group A | Group B | Group C | Group ... |
+| :-- | :-- | :-- | :-- |
+| Country1 | Example | Example | Example |
+| Country2 | Example | Example | Example |
+| Country3 | Example | Example | Example |
+| Country4 | Example | Example | Example |
+
+![](img/bracket.png "{SHORT_TITLE} bracket") <!-- remove if not available -->
+
+<!-- add the following section, once your tournament is concluded
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") |  |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") |  |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") |  |
+
+-->
+
+<!-- if the podium is an image, use the following version:
+
+## Podium
+
+![](img/podium.png "{SHORT_TITLE} podium")
+
+-->
+
+## Mappools
+
+### Stage
+
+**[Download the mappack here! (SIZE)](LINK)**
+
+- ModType
+  - [BeatmapArtist - BeatmapTitle (BeatmapCreator) [BeatmapDifficulty]](BeatmapLink)
+- Tiebreaker
+  - **Beatmap**
+
+<!-- tiebreaker beatmaps must be bolded -->
+
+## Match results
+
+### Stage
+
+Day, date: <!-- e.g. Saturday, 17 June 2018: -->
+
+<!-- For solo tournaments, replace table header with: | Player 1 |  |  | Player 2 | Match link | -->
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **WINNER** ![][flag] | **SCORE** | SCORE | ![][flag] LOSER | [#1](MatchLink) |
+| LOSER ![][flag] | -1 <!-- It's convention to write "-1" for forfeits, but this isn't required --> | **0** | ![][flag] **WINNER** | *win by default* |
+| TEAM_A ![][flag] | 0 | 0 | ![][flag] TEAM_B | *nullified* |
+
+## Ruleset
```

Comment:
I find it helpful to have a small list of possible subsections for rules like in #2796